nightfall-deployer/contracts/Shield.sol

```diff
@@ -58,12 +58,12 @@ contract Shield is Stateful, Structures, Config, Key_Registry, ReentrancyGuardUp
         uint256 time = state.getBlockData(blockNumberL2).time;
         require(
             time + COOLING_OFF_PERIOD < block.timestamp,
-            'It is too soon to get paid for this block'
+            "It is too soon to get paid for this block"
```

Comment:
I understand that both single and double quotes are acceptable for strings. 
Is there a reason why this change is required?

Reply:
Both solhint and slither shows warnings using single quotes for strings. Also I reviewed openzeppelin contracts and they always use double quotes. To follow this conventions.